easybuild/easyconfigs/p/pymatgen/pymatgen-3.5.0-intel-2016.02-GCC-4.9-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "PythonPackage"
+
+name = 'pymatgen'
+version = '3.5.0'
+
+homepage = 'https://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+
+# source package not (yet?) in url PYPI_SOURCE 
+#source_urls = [PYPI_SOURCE]
+source_urls = ['http://pypi.python.org/packages/7a/ae/c1bbe01227514bec203934d608c7319fec173791b3c58afa8e169affb976/']
```

Comment:
@migueldiascosta urgh, that's weird...

Visiting https://pypi.python.org/packages/source/p/pymatgen/pymatgen-3.5.0.tar.gz indeed gives problems:

```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>source/p/pymatgen/pymatgen-3.5.0.tar.gz</Key>
<RequestId>C4F8846AEAC375B1</RequestId>
<HostId>
fcO+45QYAedLH/AnHva6aN6ZHATgbfiYx3NcHMZFhhFPYYOBcnsefV6ZHBWt1Wzc0cnY6eifCrs=
</HostId>
</Error>
```

But, this issue is probably temporary?